AdaptiveCards/sdk/rendering-cards/xamarin/android/getting-started.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Xamarin.Android SDK
+author: almedina-ms
+ms.author: almedina
+ms.date: 12/02/2019
+ms.topic: article
+---
+
+# Getting started - Xamarin.Android
+
+This is a renderer library which targets native xamarin android applications and is based on the Android renderer that you can find [here](../../android/getting-started.md). 
+
+## NuGet install
+
+[![Nuget install](https://img.shields.io/nuget/vpre/AdaptiveCards.Rendering.Xamarin.Android.svg)](https://www.nuget.org/packages/AdaptiveCards.Rendering.Xamarin.Android)
+
+You can find the published packages [here](http://nuget.org)
+
+```console
+Install-Package AdaptiveCards.Rendering.Xamarin.Android -Version 0.1.0-alpha
+```
+
```

Comment:
You should also include what namespace imports are necessary to use the package.